### ISSUE TITLE
Xcode 11/Catalina: error: conflicting types for 'strnlen'

### ISSUE THREAD

**Issue by yurikoles** [openzfsonosx/spl#21]
[Full log](https://gist.github.com/yurikoles/198cc185bd2d9c7f33542f89beefc691)
I saw this error with Xcode 11, but now xcode-select is set to stable Xcode 10.3, but I was able to compile it this way before. Setting to Xcode 11 doesn't resolve this issue.
```
$ xcode-select -p
/Applications/Xcode.app/Contents/Developer
```

**Comment by lundman** [comment]
Unfortunately, I believe Apple has an issue in their header `kcdata.h:1246`, and I'm not sure how we should work around it.

**Comment by yurikoles** [comment]
Fixed in 45a67cae05ef4e2494f674fa9893d45b05e43667.
